monitoring/vector/vector.toml

```diff
@@ -32,6 +33,10 @@
     if exists(.log) {
       . = parse_json(string!(.log)) ?? .log
     }
+    if !exists(.message) && !exists(.log) {
+      .message = .
+    }
+
```

Comment:
interesting, do chain logs have a log file extension?

Reply:
nah these are not file extensions just fields in the JSON. without this the log messages wouldn't show up because chain logs are not in JSON. this adds the entire log line as the log message if it's not in the message or log field.